aixplain/factories/agent_factory/__init__.py

```diff
@@ -50,8 +50,44 @@ def create(
         api_key: Text = config.TEAM_API_KEY,
         supplier: Union[Dict, Text, Supplier, int] = "aiXplain",
         version: Optional[Text] = None,
+        use_supervisor: bool = True,
+        supervisor_llm_id: Optional[Text] = None,
+        use_planner: bool = False,
+        planner_llm_id: Optional[Text] = None,
     ) -> Agent:
-        """Create a new agent in the platform."""
+        """Create a new agent in the platform.
+
+        Args:
+            name (Text): name of the agent
+            llm_id (Text): aiXplain ID of the large language model to be used as agent.
+            tools (List[Tool], optional): list of tool for the agent. Defaults to [].
+            description (Text, optional): description of the agent role. Defaults to "".
+            api_key (Text, optional): team/user API key. Defaults to config.TEAM_API_KEY.
+            supplier (Union[Dict, Text, Supplier, int], optional): owner of the agent. Defaults to "aiXplain".
+            version (Optional[Text], optional): version of the agent. Defaults to None.
+            use_supervisor (bool, optional): flag to enable a multi-agent supervisor. Defaults to True.
+            supervisor_llm_id (Optional[Text], optional): aiXplain ID of the large language model to be used as the supervisor. Defaults to None.
```

Comment:
From what we discussed, I understood that we would not give the user the option to choose supervisor or planner llm id, we would just have this option internally in the communication between SDK and backend.

Reply:
Removed